src/Views/Pages/Home/Sections/Section3/index.tsx

```diff
@@ -372,6 +374,18 @@ const projects = [
 		starsPage: 'https://github.com/openscilab/dmeta/stargazers',
 		description: `DMeta is an open source Python package that removes metadata of Microsoft Office files.`,
 	},
+	{
+		title: 'MyButton',
+		repoName: 'mybutton',
+		dark_logo: mybuttonDark,
+		light_logo: mybuttonLight,
+		account: 'openscilab',
+		github: 'https://github.com/openscilab/mybutton',
+		webLink: 'https://github.com/openscilab/mybutton',
```

Comment:
I suggest to replace it with the URL of MyButton website: `https://mybutton.click`

Reply:
You are right, my bad.
Thanks